python/pyspark/errors/error_classes.py

```diff
@@ -24,6 +24,21 @@
       "Argument `<arg_name>` is required when <condition>."
     ]
   },
+  "CANNOT_ACCESS_TO_DUNDER": {
```

Comment:
We can update it as a follow-up if there is any better suggestions for the name of error classes (It's not a API changes but only affect to error message, so it would be fine)

Which one do you prefer `CANNOT_ACCESS_DUNDER` or `ILLEGAL_ACCESS_TO_DUNDER`? Can you pick one of them, or even better idea? Then I can make a follow-up for you.

Reply:
Looking at the error classes, I see a few that could use a little tweaking in the names or message wording. How about I submit a PR making several tweaks at once and tag you for review?

Reply:
> How about I submit a PR making several tweaks at once and tag you for review?

Sure! Please go ahead 👍